src/sentry/static/sentry/app/views/settings/project/projectEnvironments.jsx

```diff
@@ -241,10 +238,21 @@ class EnvironmentRow extends React.Component {
             </div>
           )}
         </Access>
-      </PanelItem>
+      </EnvironmentItem>
     );
   }
 }
+
+const EnvironmentItem = styled(PanelItem)`
```

Comment:
believe you added this const here, because I wanted to add the name `EnvironmentItem`, making it more readable...I understand you, but I'm not so sure that we should do it this way...

Reply:
Well I also added the `align-item` and `justify-content props, which I removed from this instance of `PanelItem`s props, since we will remove those in the near future.

Reply:
> since we will remove those in the near future. 

I didn't know about it ...so it make sense